lms/djangoapps/email_marketing/tasks.py

```diff
@@ -75,6 +75,10 @@ def update_user(self, sailthru_vars, email, site=None, new_user=False, activatio
     if not email_config.enabled:
         return
 
+    # do not add user if registered at a white label site
+    if not is_default_site(site):
+        return
+
```

Comment:
by moving the check for default site here, this bypasses the below call of 
```
sailthru_response = sailthru_client.api_post("user",
                                                      _create_email_user_param(sailthru_vars, sailthru_client,
                                                                               email, new_user, email_config,
                                                                               site=site))
```
Does this prevent the white label learner from getting an initial email when they register for a course? Or is this doing something different? We still want them to get registration email.

Reply:
No this does not prevent white label learner from getting registration email. That call actually creates a record in sailthru's mailing list for newly registered user.